src/GraphEditor.jsx

```diff
@@ -113,7 +126,15 @@ export default class GraphEditor extends React.Component {
                                         gCobbDouglasK={this.props.gCobbDouglasK}
                                         gCobbDouglasKName={this.props.gCobbDouglasKName}
                                         gCobbDouglasAlpha={this.props.gCobbDouglasAlpha}
+                                        gLinearDemandSupplyAUCArea={this.handleGLinearDemandSupplyAUCArea}
```

Comment:
This naming could be clearer here. Using the conventions I'm using on this project, it would be more like:

```
handleAreaUpdate={this.handleAreaUpdate}
```

Reply:
For the state values, and I suppose this applies here too, I grouped them because we'll have multiple graphs that will have AUC. Do you think these should be generic for any AUC graph, or scoped to each one?

Reply:
These should be generic - we currently have two graphs that use the AUC feature, and they can share this update function, any feature graphs that use it can share it as well, so just `handleAreaUpdate` works.